scripts/container_run.sh

```diff
@@ -8,7 +8,9 @@ readonly DB_PASS=${DB_PASS:-root}
 readonly MINIKUBE_IP=${MINIKUBE_IP:-$(minikube ip)}
 readonly SERVERNAME=${SERVERNAME:-ota.ce}
 readonly DNS_NAME=${DNS_NAME:-ota.local}
-readonly API=${API:-"http://${MINIKUBE_IP}/api"}
+readonly API_BASE_URL=${API:-"http://${MINIKUBE_IP}"}
+readonly VAULT_API=${VAULT_API:-"${API_BASE_URL}/v1"}
```

Comment:
How is the routing done if it is the same IP? Through the path?

Reply:
`Host` header and path